HW_Lesson_2/HW_Task_5.py

```diff
@@ -1 +1,9 @@
+vip_seats = ("Messi", "Ronaldo", "Zidane", "Totti", "Pele")
+common_room = ("Cafu", "Shevchenko", "Henry", None, None)
```

Comment:
leave it as a list. Only VIP are fixed. Here Nones cant be changed because you change it to a tuple